File: ORD.NET.Ui/src/js/order.ts
export default class Order {
    idOrdinazione: number;
    data: Date;
    gruppo: number;
    utenteName: string;
    zeppelinID?: number;
    piatto?: string;
    shottini: boolean;

    constructor(idOrdinazione: number,
        data: Date,
        gruppo: number,
        utenteName: string,
        shottini: boolean,
        zeppelinID?: number,
        piatto?: string) {
        this.idOrdinazione = idOrdinazione;
        this.data = data;
        this.gruppo = gruppo;
        this.utenteName = utenteName;
        this.zeppelinID = zeppelinID;
        this.piatto = piatto;
        this.shottini = shottini;
    }

}

File: ORD.NET.Model/Business/Order.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Runtime.CompilerServices;

namespace ORD.NET.Model.Business
{
    public class Order
    {
        public DateTime Data { get; set; }

        public TimeSpan OraOrdinazione { get; set; }

        public int Gruppo { get; set; }

        public User Utente { get; set; }

        public Zeppelin Zeppelin { get; set; }

        public DishType TipoPiatto { get; set; }

        public string Piatto { get; set; }

        public bool Shottini { get; set; }

        public int IdOrdinazione { get; set; }
    }
}
